README.md

```diff
@@ -1 +1,38 @@
-# angular-multiple-controllers-lab
\ No newline at end of file
+# Multiple Controllers
+
+## Objectives
+
+- Write a new Controller
+- Nest the second Controller inside the other in the View
+- Access inherited properties from the parent Controller
+- Update a parent Controller from the child Controller
+
+## Instructions
+
+Create your directory structure as follows:
+
+- js/
+  - app/
+    - controllers/
+      - ContactController.js
+      - StaffController.js
+  - angular.js
+- index.html
+
+Create the controller `ContactController` inside `js/app/controllers/ContactController.js`. Create the values `name`, `email` and `phone` inside the controller. Create a function to update the name to a different value - e.g.
+
+```js
+function ContactController() {
+  var vm = this;
```

Comment:
can you explain why you need to create `vm` and setting it equal to  `this`? That might be confusing